libraries/AP_Math/matrix_alg.cpp

```diff
@@ -405,12 +405,6 @@ bool inverse4x4(float m[],float invOut[])
 
     det = m[0] * inv[0] + m[1] * inv[4] + m[2] * inv[8] + m[3] * inv[12];
 
-#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
```

Comment:
Do we have to move the re-enabling of `FE_OVERFLOW` down as well to get things working on MacOSX?

Reply:
Yes, there was another failure detected below.